app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+from __future__ import annotations
```

Comment:
The import `from __future__ import annotations` is not necessary unless you are using forward references in type hints. If you don't need it, you can remove this line.